### PR TITLE
Add the MK80 pistol magazine to the Dropship Crew Weapons vendor

### PR DESCRIPTION
**Pull request by mnemosynium**
## About the PR
* Adds the MK80 pistol magazine to the Dropship Crew Weapons vendor

## Why / Balance
Parity

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added MK80 pistol magazines to the dropship crew weapons vendor

### CODE REVIEW

**Comment by Dutch-VanDerLinde** [comment]
THIS. IS. NOT. PARITY.

**Comment by mnemosynium** [comment]
Feel free to close it as not-parity, but you're perfectly capable of communicating that like an adult.

**Comment by QuietlyWhisper** [comment]
Its an inside joke with the community, not intended to be taken offensively,

but anyway, we'll check this out later and see what is parity again, and if this is not, consider if its worth changing.

I havent checked yet but the changes are a bit strange, could you elaborate which vendors youre trying to match them with?

**Comment by mnemosynium** [comment]
For the MK80, I’m checking the medic, engineer, and SL vendors, which offer it only in a case, not standalone. The Command vendor is the only one with the MK80 sold individually, and unlike the dropship crew vendor, it also stocks MK80 magazines.

I added the SU6 under Restricted Firearms to align with the setup in other vendors, though I wasn’t fully certain about it.

For M44 ammo, the equipment vendor sells it for 10 points, right next to a weapon vendor offering it for free. The dropship crew vendor is the only vendor that charges for standard M44 ammo.

**Comment by mnemosynium** [comment]
I've changed this PR to only replace the standard M44 speedloader with MK80 magazines, maintaining near-parity.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
This is meant to be in the weapons rack, not the equipment one, this one should keep its m44 ammo.
CMMagazinePistolVP78 should be the one getting replaced with CMMagazinePistolMK80 as thats our equivalent

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/auxiliary.yml [hunk 1]
```diff
@@ -216,7 +216,8 @@
        # amount: 14
        #- id: CMMagazinePistolMK80HP # TODO RMC14 Pistol AP varieties
        # amount: 14
-       #- id: CMMagazinePistolVP78 # Same rounds as the M1984 but I'm not sure why a different pistols mag is in here.
+       - id: CMMagazinePistolMK80
+         amount: 14
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/auxiliary.yml:220]: From 13 these are 16, though the other two commented ones are also incorrect, but we can fix that later
```suggestion
         amount: 16
```

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**